_posts/2017-07-29-css-grids.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+author: Cyan/Jos
+title: Grid CSS
+---
+
+### What is CSS Grids?
+CSS Grids is a new standard in CSS3 and is slowly being adopted by all web browsers.
+Creating a basic website like this one requires a few lines of CSS, for example:
+
+![Example Site]({{ site.url }}/assets/css-grids/example-site.png)
+
+And the CSS:
+
+```css
+.wrap {
+    border: 5px solid red;
+    height: 400px;
+}
+
+.main-content {
+    width: 45%;
+    margin-left: 1%;
+    height: 98%;
+    float: left;
+    background-color: aqua;
+}
+
+.sidebar {
+    width: 45%;
+    margin-right: 1%;
+    height: 98%;
+    float: right;
+    background-color: crimson;
+}
+```
+
+---
+
+Grid creates, well, a grid inside a div element (Think of lines between the space in the image below). 
+
+![Grid Layout]({{ site.url }}/assets/css-grids/grid-layout.png)
+
+---
+
+We can decide how many rows and columns we create as well as the gaps between them.
+After this any child elements (we cant place any child of child elements, that would be too inception-y) can be formed to fit inside this grid, which eliminates thinking about how to design a site using dozens of wrappers and placing elements inside boxes inside boxes.
```

Comment:
Comma:
> After this **,**

This next parenthetical feels misplaced and leaves me asking some questions:
> (we cant place any child of child elements, that would be too inception-y)

Perhaps add another sentence explaining why child-of-child elements are either a bad practice or actually just aren't possible (it's not clear what you mean in that regard). 

Again, avoid the informality of stuff like "inception-y" that may be lost on some readers.